WORKSPACE

```diff
@@ -3,10 +3,13 @@ workspace(name = "build_bazel_rules_ios")
 load(
     "//rules:repositories.bzl",
     "rules_ios_dependencies",
+    "rules_ios_xchammer_dependencies",
 )
 
 rules_ios_dependencies()
```

Comment:
What do you think about a parameter to this so the discoverability is a bit better? Something like `include_xchammer = True` or similar?

Reply:
I considered something like that but I've never seen this pattern being used in other rules sets? I could be wrong and it's a common thing to do 

Alternative is to update the `README` and mention this?

Reply:
`rules_xcodeproj` has a pattern similar (but for a admittedly different purpose):

https://github.com/MobileNativeFoundation/rules_xcodeproj/blob/3a8f83960c11dd9ad5e1cd5e45c53510c32cf3a0/xcodeproj/repositories.bzl#L86-L89

Same with `rules_js`, sorta:

https://github.com/aspect-build/rules_js/blob/1fcd70fc6dee2f5b4050f151846010cce4e013c3/npm/npm_import.bzl#L65-L73

Nice for it to be a single API, IMO. Probably still worth mentioning in the README, the error if it's not present, and/or the xcodeproj macro's attribute to enable xchammer support.